### DISCUSSION TITLE
串联推理时报错 FileNotFoundError: [Errno 2] No such file or directory: './ppocr/utils/ppocr_keys_v1.txt'

### DISCUSSION THREAD

**Discussion by 924553350** [PaddlePaddle/PaddleOCR#12363] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle： 2.4.0 PaddleOCR：2.6.0 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：
- FileNotFoundError                         Traceback (most recent call last)
~/PaddleOCR-2.6.0/tools/infer/predict_system.py in <module>
    232             p.wait()
    233     else:
--> 234         main(args)

~/PaddleOCR-2.6.0/tools/infer/predict_system.py in main(args)
    135     image_file_list = get_image_file_list(args.image_dir)
    136     image_file_list = image_file_list[args.process_id::args.total_process_num]
--> 137     text_sys = TextSystem(args)
    138     is_visualize = True
    139     font_path = args.vis_font_path

~/PaddleOCR-2.6.0/tools/infer/predict_system.py in __init__(self, args)
     45 
     46         self.text_detector = predict_det.TextDetector(args)
---> 47         self.text_recognizer = predict_rec.TextRecognizer(args)
     48         self.use_angle_cls = args.use_angle_cls
     49         self.drop_score = args.drop_score

~/PaddleOCR-2.6.0/tools/infer/predict_rec.py in __init__(self, args)
    101                 "rm_symbol": True
    102             }
--> 103         self.postprocess_op = build_post_process(postprocess_params)
    104         self.predictor, self.input_tensor, self.output_tensors, self.config = \
    105             utility.create_predictor(args, 'rec', logger)

~/PaddleOCR-2.6.0/ppocr/postprocess/__init__.py in build_post_process(config, global_config)
     64     assert module_name in support_dict, Exception(
     65         'post process only support {}'.format(support_dict))
---> 66     module_class = eval(module_name)(**config)
     67     return module_class

~/PaddleOCR-2.6.0/ppocr/postprocess/rec_postprocess.py in __init__(self, character_dict_path, use_space_char, **kwargs)
    112                  **kwargs):
    113         super(CTCLabelDecode, self).__init__(character_dict_path,
--> 114                                              use_space_char)
    115 
    116     def __call__(self, preds, label=None, *args, **kwargs):

~/PaddleOCR-2.6.0/ppocr/postprocess/rec_postprocess.py in __init__(self, character_dict_path, use_space_char)
     31             dict_character = list(self.character_str)
     32         else:
---> 33             with open(character_dict_path, "rb") as fin:
     34                 lines = fin.readlines()
     35                 for line in lines:

FileNotFoundError: [Errno 2] No such file or directory: './ppocr/utils/ppocr_keys_v1.txt'

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.
![屏幕截图 2024-02-02 135354](https://github.com/PaddlePaddle/PaddleOCR/assets/133498191/86e352eb-0b36-497f-b29f-e700e2315a50)
![1](https://github.com/PaddlePaddle/PaddleOCR/assets/133498191/a9b05f13-bef6-4bf0-bae1-5d730571dc8e)

**Comment by 924553350** [comment]
https://aistudio.baidu.com/projectdetail/5766320

**Comment by a31413510** [comment]
看看你的这个路径下，有没有这个文件呢，/ppocr/utils/ppocr_keys_v1.txt

**Comment by 924553350** [comment]
> 看看你的这个路径下，有没有这个文件呢，/ppocr/utils/ppocr_keys_v1.txt

这个文件的路径是/home/aistudio/PaddleOCR-2.6.0/ppocr/utils/ppocr_keys_v1.txt
